api/OfflineAudioContext.json

```diff
@@ -270,7 +270,7 @@
             "opera": "mirror",
             "opera_android": "mirror",
             "safari": {
-              "version_added": "6"
+              "version_added": "7"
```

Comment:
Without `startRendering` this API doesn't work at all. Is this really correct?

Reply:
Looks like the entire API isn't supported until Safari 7, updating now!

Reply:
Alright, so the custom tests here aren't totally in sync, the api.OfflineAudioContext test is just `'OfflineAudioContext' in self`.